plugins/filesystem.go

```diff
@@ -46,6 +50,7 @@ func (p *FileSystemPlugin) DefineCommand(channels Channels) (*cobra.Command, err
 	}
 
 	flags.StringSliceVar(&p.Ignored, flagIgnored, []string{}, "Patterns to ignore")
+	flags.StringVar(&p.ProjectName, flagProjectName, "", "Project name to differentiate between filesystem scans")
```

Comment:
optional or mandatory? best ux is for this to be optional

Reply:
Optional. It will be mandatory only if I will add `flags.MarkFlagRequired(flagProjectName)`